store/sqlstore/upgrade.go

```diff
@@ -1,815 +0,0 @@
-		return false
-	}
-
-	currentVersion, err := semver.Parse(currentSchemaVersion)
-	if err != nil {
-		mlog.Error("Error parsing current schema version", mlog.Err(err))
-		return false
-	}
-
-	if storedVersion.Major == currentVersion.Major && storedVersion.Minor == currentVersion.Minor {
-		mlog.Warn("Attempting to upgrade the database schema version",
-			mlog.String("stored_version", storedSchemaVersion), mlog.String("current_version", currentSchemaVersion), mlog.String("new_version", expectedSchemaVersion))
-		return true
-	}
-
-	return false
-}
-
-func upgradeDatabaseToVersion31(sqlStore *SqlStore) {
-	if shouldPerformUpgrade(sqlStore, Version300, Version310) {
-		saveSchemaVersion(sqlStore, Version310)
-	}
-}
-
-func upgradeDatabaseToVersion32(sqlStore *SqlStore) {
-	if shouldPerformUpgrade(sqlStore, Version310, Version320) {
-		saveSchemaVersion(sqlStore, Version320)
-	}
-}
-
-func themeMigrationFailed(err error) {
-	mlog.Fatal("Failed to migrate User.ThemeProps to Preferences table", mlog.Err(err))
-}
-
-func upgradeDatabaseToVersion33(sqlStore *SqlStore) {
-	if shouldPerformUpgrade(sqlStore, Version320, Version330) {
-		if sqlStore.DoesColumnExist("Users", "ThemeProps") {
-			transaction, err := sqlStore.GetMasterX().Beginx()
-			if err != nil {
-				themeMigrationFailed(err)
-			}
-			defer finalizeTransactionX(transaction)
-
-			// copy data across
-			if _, err := transaction.ExecNoTimeout(
-				`INSERT INTO
-					Preferences(UserId, Category, Name, Value)
-				SELECT
-					Id, '` + model.PreferenceCategoryTheme + `', '', ThemeProps
-				FROM
-					Users
-				WHERE
-					Users.ThemeProps != 'null'`); err != nil {
-				themeMigrationFailed(err)
-				return
-			}
-
-			// delete old data
-			if _, err := transaction.ExecNoTimeout("ALTER TABLE Users DROP COLUMN ThemeProps"); err != nil {
-				themeMigrationFailed(err)
-				return
-			}
-
-			if err := transaction.Commit(); err != nil {
-				themeMigrationFailed(err)
-				return
-			}
-		}
-
-		saveSchemaVersion(sqlStore, Version330)
-	}
-}
-
-func upgradeDatabaseToVersion34(sqlStore *SqlStore) {
-	if shouldPerformUpgrade(sqlStore, Version330, Version340) {
-		saveSchemaVersion(sqlStore, Version340)
-	}
-}
-
-func upgradeDatabaseToVersion35(sqlStore *SqlStore) {
-	if shouldPerformUpgrade(sqlStore, Version340, Version350) {
-		sqlStore.GetMasterX().Exec("UPDATE TeamMembers SET Roles = 'team_user' WHERE Roles = ''")
-		sqlStore.GetMasterX().Exec("UPDATE TeamMembers SET Roles = 'team_user team_admin' WHERE Roles = 'admin'")
-		sqlStore.GetMasterX().Exec("UPDATE ChannelMembers SET Roles = 'channel_user' WHERE Roles = ''")
-		sqlStore.GetMasterX().Exec("UPDATE ChannelMembers SET Roles = 'channel_user channel_admin' WHERE Roles = 'admin'")
-
-		// The rest of the migration from Filenames -> FileIds is done lazily in api.GetFileInfosForPost
-		sqlStore.CreateColumnIfNotExists("Posts", "FileIds", "varchar(150)", "varchar(150)", "[]")
```

Comment:
This is not needed because `FileIds` is already varchar(300) and there is a migration for that.